scripts/paper_evaluations.yml

```
@@ -74,16 +74,25 @@ training_checkpoint_sweep:
 
 # Experiment: evaluate adversary vs. KataGo checkpoints.
 katago_ckpt_sweep:
-  num_games_per_matchup: 50
+  # Set adversary_path to null to use the main adversary.
+  adversary_path: "/nas/ucb/k8/go-attack/victimplay/\
+    ttseng-cyclic-vs-cp564-20230213-154321/models/\
+    t0-s22887680-d6258166/model.bin.gz"  # cyclic-adv-ft-vs-b60-s7047m-s22m
+  num_games_per_matchup: 500
   # Adversary config
   adversary_algorithm: AMCTS
   adversary_visits: 600
   # Victim config
-  victim_visits: 32
+  victim_visits:
+    - 1
+    - 32
+    - 200
+  net_sizes:
+    - b60
   victim_dir: /nas/ucb/k8/go-attack/katago-networks
   # We evaluate victims newer or equal to this checkpoint.
   # We compute victim order using the d value.
-  victim_start: kata1-b40c256-s11840935168-d2898845681 # cp505
+  victim_start: kata1-b60c320-s7047906048-d3140270330
   # Pick the max number of victims that can fit on a GPU.
```

```suggestion
  # The number of victims per GPU. Be careful not to exceed GPU memory limits.
```

Just want to reword this line slightly so as to not imply that we're selecting this value solely based on GPU memory limits.  If we have several values of `victim_visits`, we can fit more victims in one job while staying within GPU memory limits due to victims sharing the same network file. We're probably more limited by keeping `numGamesTotal` at a low enough value.